app/src/main/java/tech/relaycorp/courier/data/model/StoredMessage.kt

```diff
@@ -0,0 +1,26 @@
+package tech.relaycorp.courier.data.model
+
+import androidx.room.Entity
+import androidx.room.Index
+import java.util.Date
+
+@Entity(
+    tableName = "Message",
+    primaryKeys = ["senderAddress", "messageId"],
+    indices = [
+        Index(value = ["recipientAddress"]),
+        Index(value = ["messageType"]),
+        Index(value = ["expirationTime"])
+    ]
+)
+data class StoredMessage(
+    val recipientAddress: MessageAddress,
+    val recipientType: MessageAddress.Type,
+    val senderAddress: MessageAddress,
+    val messageId: MessageId,
+    val messageType: MessageType,
+    val creationTime: Date, // in UTC
+    val expirationTime: Date, // in UTC
```

Comment:
I realise this code isn't new, but I just thought it'd be good to add the "UTC" suffix instead of relying on the comment as that wouldn't be visible from any code using that field.

I think this is important now that we're using `Date` objects and I'm planning on making the same change to the core JVM library when I get round to replacing `ZonedDateTime` with `Date`.

Reply:
👍